### DISCUSSION TITLE
Zigpy version in HA + most recent tag

### DISCUSSION THREAD

**Discussion by mdeweerd** [zigpy/zigpy#914] [ZHA development]
I was trying to use zigpy.foundation.GeneralCommands in HA2022.2.3 to find that it does not exist there and that HA is actually using 0.43.0 which is also the latest tag for zigpy .

While zigpy.foundation.GeneralCommands was renamed from zigpy.foundation.Commands since march last year, I am surprised it's not yet in HA.  Possibly because there 0.43.0 is the last tag.

In the mean time, zigpy-znp (and others) are updated regularly.

So I wonder about consistency issues that might currently go undiscovered.

**Comment by Adminiuga** [comment]
When was the PR merged? That is what matters, not when the PR was 1st authored.

**Reply by mdeweerd** [reply]
https://github.com/zigpy/zigpy/pull/716 - so just a few days ago.  I did not think of such a long delay ;-).

**Reply by Hedda** [reply]
https://github.com/zigpy/zigpy/pull/716 was merged just 4-days ago so it is not even in any new zigpy release yet ;-)

https://github.com/zigpy/zigpy/commits/dev

https://github.com/zigpy/zigpy/releases

ZHA dependencies does seem to get bumped regularly however there is still a delay since might not get added to minor releases of HA. 

https://github.com/home-assistant/core/pulls?q=is%3Apr+is%3Aclosed+zha+OR+zigpy

Also guessing that ZHA/zigpy developers will try to avoid updating zigpy for ZHA in between HA releases unless only contain a small bug-fix?

That way there is at least a chance for Home Assistant beta-testers to test it before it is released as part of a new xxxx.x.0 general release.

https://www.home-assistant.io/faq/release/

And if https://github.com/zigpy/zigpy/pull/716 is a major update it should probably not be bumped as part of a minor release of zigpy or as a minor release of HA? :-)
